### DISCUSSION TITLE
Scrolling bug on 12.2 with trackpad

### DISCUSSION THREAD

**Discussion by GES-YO** [pilotmoon/Scroll-Reverser#138] [Q&A] [answered]
After updating to 12.2 when scrolling down the page scrolls and immediately returns to top. This happens every time you scroll down. The page cannot be scrolled.

**Comment by craftey** [comment]
https://github.com/pilotmoon/Scroll-Reverser/issues/136

**Comment by pilotmoon** [comment, marked as answer]
Now fixed, in v1.8.2.

**Reply by NODA5** [reply]
Still broken after updating!

**Reply by Firehed** [reply]
Same - experiencing the same issue on macOS 12.3 & ScrollReverser 1.8.2. I have tried relaunching the app and the issue persists.
